kubernetes/helm_charts/core/nginx-private-ingress/templates/configmap.yaml

```diff
@@ -174,17 +174,17 @@ data:
         }
         location /gql-service/ {
           set $target http://gql-service.{{ .Values.namespace }}.svc.cluster.local:8080;
-          rewrite ^/gql-service/(.*) /$1 break;
+          rewrite ^/(.*) /$1 break;
```

Comment:
I don't think we need this rewrite block, as we're replacing the same string. like, replace apple with apple. So it'll be good to drop all these rewrites.

Reply:
Great thanks. Will just update this.